sekoia_automation/module.py

```diff
@@ -269,6 +269,11 @@ def register(self, item: type["ModuleItem"], name: str = ""):
             item.name = name
         self._items[name] = item
 
+    def register_account_validator(self):
+        from sekoia_automation.account_validator import AccountValidator
+
+        self.register(AccountValidator, "validate_module_configuration")
```

Comment:
The creator of the module must be able to specify its own implementation of the validator

```suggestion
    def register_account_validator(self, validator: type["AccountValidator"] ):
        self.register(validator, "validate_module_configuration")
```